demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py

```diff
@@ -265,6 +266,45 @@ def _add_all_level_dependencies(
                     ),
                 )
 
+    def _add_all_level_imports(
```

Comment:
you don't need a seperate function for this, this is very very similar to `_add_level_dependencies`.

Reply:
combined and rename _add_all_level_relationships